test/WebJobs.Script.Tests.Integration/NodeEndToEndTests.cs

```diff
@@ -1229,6 +1229,12 @@ public void ExcludedFunction_NotAddedToHost()
             Assert.Equal(1, array[1]);
         }
 
+        public void Dispose()
+        {
+            // Make sure all logs are flushed before next test.
```

Comment:
Check out the TestHelpers.GetLogs call, it has a built in delay (the logger flush interval) - perhaps boost that? If we could localize it that would probably be best though to minimize the time increases.